packages/quantic/typings/lwc/engine.d.ts

```diff
@@ -126,6 +126,20 @@ declare module 'lwc' {
         compareDocumentPosition(otherNode: Node): number;
         contains(otherNode: Node): boolean;
         elementFromPoint(x: number, y: number): Element | null;
+        querySelector<K extends keyof HTMLElementTagNameMap>(
```

Comment:
calling querySelector without ts-ignore was raising typing errors. Upon investigation I noticed salesforce has updated their LWC typings since we first imported them into Quantic. So I updated our typings from [lwc repo](https://github.com/salesforce/lwc/blob/5100fc0b3db3238ae97316c494d06fe328c6bf1f/packages/lwc/types.d.ts#L129)

Reply:
Thank you for the fix!